app/src/main/res/layout/fragment_onboarding_pager.xml

```diff
@@ -40,8 +40,9 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
-                android:background="@android:color/transparent"
+                android:background="?attr/paper_color"
```

Comment:
@cooltey  The issue was already fixed, and applying `paper_color` here is unnecessary.